images/manageiq-base/Dockerfile

```diff
@@ -19,17 +12,6 @@ ARG HOOKS_SCRIPT_URL
 
 RUN dnf -y --disableplugin=subscription-manager --setopt=tsflags=nodocs install git
 
-RUN mkdir appliance && \
-    if [[ -n "$GIT_AUTH" ]]; then GIT_HOST=${GIT_AUTH}@${GIT_HOST}; fi && curl -L https://${GIT_HOST}/${APPLIANCE_ORG}/${CORE_REPO_NAME}-appliance/tarball/${APPLIANCE_REF} | tar vxz -C appliance --strip 1
-
-RUN mkdir sui && \
-    if [[ -n "$GIT_AUTH" ]]; then GIT_HOST=${GIT_AUTH}@${GIT_HOST}; fi && curl -L https://${GIT_HOST}/${SUI_ORG}/${CORE_REPO_NAME}-ui-service/tarball/${SUI_REF} | tar vxz -C sui --strip 1 && \
-    echo "`git ls-remote https://${GIT_HOST}/${SUI_ORG}/${CORE_REPO_NAME}-ui-service.git ${SUI_REF} | cut -f 1`" > sui/BUILD
-
-RUN mkdir app && \
-    if [[ -n "$GIT_AUTH" ]]; then GIT_HOST=${GIT_AUTH}@${GIT_HOST}; fi && curl -L https://${GIT_HOST}/${MIQ_ORG}/${CORE_REPO_NAME}/tarball/${MIQ_REF} | tar vxz -C app --strip 1 && \
-    echo "`date +'%Y%m%d%H%M%S'`_`git ls-remote https://${GIT_HOST}/${MIQ_ORG}/${CORE_REPO_NAME}.git ${MIQ_REF} | cut -c 1-7`" > app/BUILD
-
 RUN mkdir build && \
     if [[ -n "$GIT_AUTH" ]]; then GIT_HOST=${GIT_AUTH}@${GIT_HOST}; fi && curl -L https://${GIT_HOST}/${BUILD_ORG}/${CORE_REPO_NAME}-appliance-build/tarball/${BUILD_REF} | tar vxz -C build --strip 1
```

Comment:
We may be able to drop this entire hook now, because all of the customization should happen in the RPM layer.  cc @carbonin

Reply:
2 things are still coming outside of RPMs. I'll remove the hook when that goes away.

https://github.com/ManageIQ/manageiq-pods/pull/485/files#diff-4f1c0cb40242780de19fd44d5c011b0eL121-L124